From e427f9da3501fe3549c9cb257bb7bca9e81038ed Mon Sep 17 00:00:00 2001
From: Mahesh Mahadevan <r9aadq@freescale.com>
Date: Fri, 7 Mar 2008 10:46:47 -0600
Subject: [PATCH] ENGR00067988 Fix SDRAM address passed to SDMA script

Updated the address passed in through watermark field to the mem-2-mem
transfer script.

Signed-off-by: Mahesh Mahadevan <r9aadq@freescale.com>

 ENGR00067988 Update copyright

 Updated the address passed in through watermark field to the mem-2-mem
 transfer script.

 Signed-off-by: Mahesh Mahadevan <r9aadq@freescale.com>
---
 arch/arm/plat-mxc/sdma/sdma.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/plat-mxc/sdma/sdma.c b/arch/arm/plat-mxc/sdma/sdma.c
index 72e6895..d21c6a8 100644
--- a/arch/arm/plat-mxc/sdma/sdma.c
+++ b/arch/arm/plat-mxc/sdma/sdma.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004-2007 Freescale Semiconductor, Inc. All Rights Reserved.
+ * Copyright 2004-2008 Freescale Semiconductor, Inc. All Rights Reserved.
  */
 
 /*
@@ -40,7 +40,7 @@
 
 #include "iapi.h"
 
-#define M3_BASE_ADDRESS 0x80000000
+#define M3_BASE_ADDRESS CSD0_BASE_ADDR
 #define CHAD(ch) sdma_data[0].cd->ccb_ptr[ch].channelDescriptor
 
 /*!
-- 
1.5.4.4

